polkadot/runtime/westend/src/lib.rs

```diff
@@ -1657,11 +1657,8 @@ pub type SignedExtra = (
 );
 
 parameter_types! {
-	// This is the max pools that will be migrated in the runtime upgrade. Westend has more pools
-	// than this, but we want to emulate some non migrated pools. In prod runtimes, if weight is not
-	// a concern, it is recommended to set to (existing pools + 10) to also account for any new
-	// pools getting created before the migration is actually executed.
-	pub const MaxPoolsToMigrate: u32 = 250;
+	/// Bounding number of agent pot accounts to be migrated in a single block.
+	pub const MaxAgentsToMigrate: u32 = 300;
```

Comment:
Current count of agents (same as pools) in Westend is 269. `300` should be enough to cover all.